contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1857,6 +1857,24 @@ class tsqlBuilder : public tsqlCommonMutator
 	void exitFunction_call(TSqlParser::Function_callContext *ctx) override
 	{
 		is_function = true;
+		if (ctx->NEXT())
+		{
+			std::string seq_name;
+			if (ctx->full_object_name())
+			{
+				seq_name = ::getFullText(ctx->full_object_name());
+			}
+			else
+			{
+				elog(WARNING, "placeholder");
```

Comment:
What is this for and what happens if `ctx->full_object_name()` is null? Is it safe to proceed?